cmd/minishift/cmd/start.go

```diff
@@ -254,6 +255,17 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	autoMountHostFolders(hostVm.Driver)
 
+	// start the minishift system tray
+	if runtime.GOOS != "linux" {
+		if viper.GetBool(configCmd.AutoStartTray.Name) {
+			minishiftTray := systemtray.NewMinishiftTray(minishiftConfig.AllInstancesConfig)
+			err = minishiftTray.EnsureRunning()
+			if err != nil {
+				fmt.Println(err)
+			}
+		}
+	}
+
```

Comment:
@anjannath this should be a separate function as we should keep `func runStart` as clean as possible.